Userland/Applications/CalendarSettings/CalendarSettingsWidget.gml

```diff
@@ -36,6 +36,64 @@
         }
     }
 
+    @GUI::GroupBox {
```

Comment:
This needs to be formatted (see CI)
Easiest way to do that is `./Meta/lint-gml-format.sh`, which will modify the GML in-place.